frontend/src/router/index.ts

```diff
@@ -117,12 +117,16 @@ router.beforeEach((to, from, next) => {
 
 // SECOND update user data (requires auth)
 router.beforeEach((to, from, next) => {
-  if ('login' in to.query) {
+  if ('sync_me' in to.query) {
     authPromise.then(async auth => {
       if (auth.isAuthenticated()) {
         await backend.users.syncMe();
+        delete to.query.sync_me;
```

Comment:
is this to avoid endless recursion? please add a comment explaining it

Reply:
Exactly, fixed in https://github.com/cryptomator/hub/pull/135/commits/36007654d94be552132380fcf19c6bb563cdef9d